appservice/src/deploy/deployToStorageAccount.ts

```diff
@@ -15,13 +17,16 @@ import { SiteClient } from '../SiteClient';
 import { randomUtils } from '../utils/randomUtils';
 import { runWithZipStream } from './runWithZipStream';
 
+dayjs.extend(relativeTime);
+dayjs.extend(utc);
+
 /**
  * Method of deployment that is only intended to be used for Linux Consumption Function apps because it doesn't support kudu pushDeployment
  * To deploy with Run from Package on a Windows plan, create the app setting "WEBSITE_RUN_FROM_PACKAGE" and set it to "1".
  * Then deploy via "zipdeploy" as usual.
  */
 export async function deployToStorageAccount(context: IActionContext, fsPath: string, client: SiteClient): Promise<void> {
-    const datePart: string = moment.utc(Date.now()).format('YYYYMMDDHHmmss');
+    const datePart: string = dayjs().utc().format('YYYYMMDDHHmmss');
```

Comment:
I compared the output from the old code vs. the new code, here and both places below, to ensure the identical outcome.